packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -294,3 +297,76 @@ Uncompressed batch:
 | ClientSeqNum: 1 | ClientSeqNum: 2 | ClientSeqNum: 3 | ClientSeqNum: 4 | ClientSeqNum: 5 |
 +-----------------+-----------------+-----------------+-----------------+-----------------+
 ```
+
+## How the overall op flow works
+
+### Outbound
+
+The outbound view is how ops are accumulated and sent by the runtime with `FlushMode.TurnBased` (default).
+
+```mermaid
+stateDiagram-v2
+	state "* End of JS turn *" as jsTurn
+	state "Group batch" as opGrouping
+	state "Send batch over the wire" as post
+	state "Send chunks (partial ops) over the wire" as postChunks
```

Comment:
Is this really partial _ops_ or partial (lower level) _messages_? After compressing a batch with more than one op, I would've thought the resulting chunks might be pieces of a compressed payload that doesn't necessarily map to ops cleanly. E.g., that a given chunk might contain everything needed to reconstruct op 1 but also half of what's needed to reconstruct op 2.